packages/replicache-doc/package.json

```diff
@@ -26,6 +26,7 @@
     "react-dom": "^18.3.1"
   },
   "devDependencies": {
+    "replicache": "15.2.1",
```

Comment:
Without this dep, replicache-doc does not get rechecked/rebuilt when replicache changes